src/app.py

```diff
@@ -2856,6 +2856,14 @@ def get_prov_info():
         # Each dataset's dictionary is added to the list to be returned
         dataset_prov_list.append(internal_dict)
 
+    # Determine whether the size of the returned data exceeds or nearly exceeds the AWS Gateway 10MB maximum size. If it
+    # is greater than 9MB Return a 400 and prompt the user to reduce the size of the output by applying optional
+    # argument filters.
+    dataset_prov_json_encode = json.dumps(dataset_prov_list).encode('utf-8')
+    if len(dataset_prov_json_encode) > 9000000:
```

Comment:
Let's keep it consistent by using `9*(2**20)` (9437184) rather than `9000000` https://github.com/hubmapconsortium/uuid-api/blob/main/src/instance/app.cfg.example#L35 which is closer to 10MB. Also update the comment.